### PR TITLE
1.9.0 release

### PR DESCRIPTION
**Pull request by williamhorning**

### CODE REVIEW

File: src/lib/settings/ParentalControls.svelte
Comment (JoshAtticus) [inline src/lib/settings/ParentalControls.svelte:1]: Why does this exist? It's just empty? I thought we weren't adding parental controls to Meower with the under 13 ban

**Review by showierdata9978 — APPROVED** [review]

File: src/pages/about.svelte [hunk 1]
```diff
@@ -115,6 +115,20 @@
 </Container>
 <Container>
 	<h2>Changelog</h2>
+	<Container>
+		<h2>1.9.0</h2>
+		<ul>
+			<li>Add api accounts support</li>
```
Comment (JoshAtticus) [inline src/pages/about.svelte:121]: Change this to "Migrate account actions to API requests" or "Use API requests for account actions"

**Review by JoshAtticus — CHANGES_REQUESTED** [review]
See my review comments, otherwise looks awesome

File: src/lib/settings/ParentalControls.svelte
Comment (williamhorning) [reply src/lib/settings/ParentalControls.svelte:1]: This is most likely a remnant of when we were still debating that stuff. It’s not used anywhere so I’m cool with deleting it.

File: src/pages/about.svelte (hunk 1 above)
Comment (williamhorning) [reply src/pages/about.svelte:121]: We’ve been calling these changes api accounts everywhere else but I’ll change that

Comment (JoshAtticus) [reply src/pages/about.svelte:121]: > We’ve been calling these changes api accounts everywhere else but I’ll change that

Yeah it just seems off, and most definitely confusing to less technical users

**Review by JoshAtticus — APPROVED** [review]
Awesome

**Comment by mybearworld** [comment]
Reverted with https://github.com/meower-media-co/Meower-Svelte/pull/249

**Pull request merged**